lib/resources/Sources.js

```diff
@@ -6,7 +6,7 @@ const stripeMethod = StripeResource.method;
 module.exports = StripeResource.extend({
   path: 'sources',
 
-  includeBasic: ['create', 'retrieve', 'update', 'setMetadata', 'getMetadata'],
+  includeBasic: ['create', 'retrieve', 'update'],
 
   listSourceTransactions: stripeMethod({
```

Comment:
Same as above, it's called `listSourceTransactions` to be consistent with other nested resources. If there were other methods they'd be called `createSourceTransaction` etc. (In this case the endpoint does return a list object containing a list of `source_transaction` objects, so `listSourceTransactions` is the correct name.)

For `detach`, you're correct that it's missing, but you can call `customers.deleteSource()` which does exactly the same thing (although the name is a bit misleading since in the case of a source, it won't actually be deleted, just detached from the customer).

Reply:
That `detach` thing looks pretty weird to me (and by weird, I mean, quite probably a bug? cc @mickjermsurawong-stripe )

Reply:
Any reason that this is `listSourceTransactions` as opposed to `sourceTransactions`? Differs from OpenAPI in this regard.

Reply:
FWIW, OpenAPI also includes the following, which is not included here:

```js
detach: stripeMethod({
  method: 'DELETE',
  path: '/{id}/{id}',
  urlParams: ['id'],
}),
```